### DISCUSSION TITLE
Nested hydras?

### DISCUSSION THREAD

**Discussion by dizzib** [nvimtools/hydra.nvim#45] [Q&A]
I have defined a custom hydra called NeotestHydra which I activate for a unit testing session.

To debug a unit test I would like to activate my custom DapHydra as a nested hydra, so when I quit DapHydra, it should automatically return to an active NeotestHydra.

Is this possible?

Thanks.

**Comment by benlubas** [comment]
Potentially a usecase for [hooks](https://github.com/nvimtools/hydra.nvim?tab=readme-ov-file#hooks)
